deployment/network-operator/README.md

```diff
@@ -218,15 +218,45 @@ helm upgrade -n network-operator  network-operator mellanox/network-operator --v
 
 >__NOTE__: `--devel` option required if you want to use the beta release
 
-### Return PODs to the drained nodes
 
->__NOTE__: this operation required only if **containerized OFED** is in use and if drain command
-was applied before to remove PODs from the cluster nodes
+### Restart PODs with containerized OFED drive 
+
+>__NOTE__: this operation required only if **containerized OFED** is in use
+
+Helm upgrade command will upgrade DaemonSet of the OFED driver to point to the new driver version,
+but will not remove or restart PODs with the driver on the nodes.
+The old OFED version will run on a node until you explicitly remove the driver POD or reboot the node.
+
+
+#### Remove PODs with secondary network from the node
+When containerized OFED driver reloaded on the node, all PODs which use secondary network based on
+NVIDIA Mellanox NICs will lose network interface in their containers.
+To prevent outage you need to remove all PODs which use secondary network from the node
+before you reload the driver POD.
+
+This can be done with node drain command:
+```
+kubectl drain <NODE_NAME> --pod-selector=<selector_for_pods>
```

Comment:
can you give example on how to use this command?
It expect to run this command per node?

Reply:
Yes, this command should run per node. I will add details under another comment from you